### DISCUSSION TITLE
Useful hints for utilizing it the best

### DISCUSSION THREAD

**Discussion by AhmedAbdalgader** [community/community#123979] [GitHub Education]
Hello 
I've just got the good news of becoming GitHub student.
Seeing the student Developer Pack offers, I was wondering about the best way to boost my coding skills.
I'm a Geoinformatics student.
Anyone had a successful experience with one of the offers like;  codedex, ....

**Comment by goldieowner17** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @AhmedAbdalgader! We're happy you're here. Congrats on getting the Student Pack!

For this post, you're more likely to get a useful response if you are posting your question in the applicable category. The Discover category is a place for GitHub employees to share best practices.

Also, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!
